config.toml.sample

```diff
@@ -1,6 +1,6 @@
 [app]
     # Interface and port where the app will run its webserver.
-    address = "0.0.0.0:9000"
+    address = "0.0.0.0:443"
```

Comment:
It should not be assumed or demonstrated by default that listmonk runs on 80/443. The current example using a non privileged port by default is better. Otherwise you need to run as root, and that should not be suggested by default. I suggest reverting this back, and just document in a comment nearby the need to run on 80/443 if using the built in SSL cert features.

Also note that if using the TLS feature not one but 2 ports with be used by the app, and that should be reflected in the config somehow.